libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -749,7 +749,7 @@ void AP_TECS::_update_throttle_with_airspeed(void)
         // drag increase during turns.
         const float cosPhi = sqrtf((rotMat.a.y*rotMat.a.y) + (rotMat.b.y*rotMat.b.y));
         STEdot_dem = STEdot_dem + _rollComp * (1.0f/constrain_float(cosPhi * cosPhi, 0.1f, 1.0f) - 1.0f);
-        const float ff_throttle = nomThr + STEdot_dem / (_STEdot_max - _STEdot_min) * (_THRmaxf - _THRminf);
```

Comment:
this is not equivalent. The const var is doing / and this one is doing *.

Reply:
I did the math on my mind, with pencil and paper and here too:
```c
nomThr + STEdot_dem / (_STEdot_max - _STEdot_min) * (_THRmaxf - _THRminf) =
nomThr + STEdot_dem * (_THRmaxf - _THRminf) / (_STEdot_max - _STEdot_min) =
nomThr + STEdot_dem * ( (_THRmaxf - _THRminf) / (_STEdot_max - _STEdot_min) ) =
nomThr + STEdot_dem / ( (_STEdot_max - _STEdot_min) / (_THRmaxf - _THRminf) ) =
nomThr + STEdot_dem / K_thr2STE
```
Perhaps you can give it another look?

Reply:
Not sure why it fails the CI, though.

Reply:
Ping @magicrub